interview-questions.md

```diff
@@ -8,44 +8,44 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a class?
 
-Your answer:
+Your answer: Calling a super in a class brings down the instances of the super class down the inheritance of the sub-classes. 
 
-Researched answer:
+Researched answer: The super method calls the initialization method that is inherited from the superclass and can be used in the subclass.
 
 2. What is a gem?
 
-Your answer:
+Your answer: A gem is a package manager specifically for Ruby
 
-Researched answer:
+Researched answer: A gem is a library or a snippet of code functionality. RubyGems is the package manager used specifically for Ruby.
 
 3. What is a relational database? Are there other kinds of databases?
 
-Your answer:
+Your answer: A relational database is a database that is used to find information pertinent to your search parameters. There are other kinds of databases.
 
-Researched answer:
+Researched answer: A relational database has information structured in tables, which contain rows and columns. What makes the database relational is when the tables have relationships to each other.
```

Comment:
Make sure to address the second part of the question.  Your initial answer is very vague when referencing the second part to the question and your researched answer does not address it at all.